### PR TITLE
feat: add blog post for messaging v1.3 release

### PR DESCRIPTION
**Pull request by stijnmoreels**
Add blog post proposal for Arcus Messaging v1.3 release.

Relates to #245

### CODE REVIEW

File: community/blog/2022-09 Added EventHubs support, correlated Service Bus messages, and more in Arcus Messaging v1.3/README.md [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+        await _serviceBusSender.SendAsync(order, correlation, _logger);
+    }
+}
+```
+
+⚡ We have similar functionality for EventHubs where we provide `SendAsync` extensions on the `EventHubsProducerClient`.
+
+For more information see our official documentation for [Service Bus extensions](https://messaging.arcus-azure.net/Features/service-bus-extensions) and [EventHubs extensions](https://messaging.arcus-azure.net/Features/event-hubs-extensions).
+
+💡 We have a [user guide]() that explains fully how service-to-service correlations work for applications that integrate Service Bus.
```
Comment (stijnmoreels) [inline community/blog/2022-09 Added EventHubs support, correlated Service Bus messages, and more in Arcus Messaging v1.3/README.md:113]: Waiting for https://github.com/arcus-azure/arcus.observability/pull/467

File: community/blog/2022-09 Added EventHubs support, correlated Service Bus messages, and more in Arcus Messaging v1.3/README.md [hunk 2]
```diff
@@ -0,0 +1,129 @@
+# Added EventHubs supported, correlated Service Bus messages, and more in Arcus Messaging v1.3
+Arcus Messaging v1.3 is a big release with big changes. Full of features, extended functionality, and useful helpers. This post goes over the major parts of this release.
+
+## EventHubs message handling support
+Arcus Messaging v1.3 introduces a new type of messaging to the family: EventHubs messaging. Previously, we only supported Azure Service Bus messaging in both message pump worker and Azure Functions trigger scenarios. EventHubs support was a long-requested feature and received more priority due to our internal community. This shows that Arcus is driven by community work and not by higher-up ideas.
```
Comment (fgheysels) [inline community/blog/2022-09 Added EventHubs support, correlated Service Bus messages, and more in Arcus Messaging v1.3/README.md:5]:
```suggestion
Arcus Messaging v1.3 introduces a new type of messaging to the family: EventHubs messaging. Previously, we only supported Azure Service Bus messaging in both message pump worker and Azure Functions trigger scenarios. EventHubs support was a long awaited feature and received more priority due to our internal community. This shows that Arcus is driven by community work and not by higher-up ideas.
```

File: community/blog/2022-09 Added EventHubs support, correlated Service Bus messages, and more in Arcus Messaging v1.3/README.md [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+    .ConfigureServices(services =>
+    {
+        services.AddEventHubsMessagePump("<event-hubs-name>", "<event-hubs-connection-string-secret-name>", "<blob-container-name>", "<storage-account-connection-string-secret-name>")
+                .WithEventHubsMessageHandler<SensorReadingAzureEventHubsMessageHandler, SensorReading>();
+    });
+```
+
+⚠ Note that the EventHubs connection string and storage account connection string are not passed along directly to the message pump extension, but that a secret name is provided. This secret name should correspond with a stored secret in the Arcus secret store. For more information on the secret store, see [our official documentation](https://security.arcus-azure.net/features/secret-store).
+
+💡 The EventHubs message pump and message router have a lot more options to influence the message handling to your needs. See [our official documentation](https://messaging.arcus-azure.net/Features/message-handling/event-hubs) for more information on this and other topics.
```
Comment (fgheysels) [inline community/blog/2022-09 Added EventHubs support, correlated Service Bus messages, and more in Arcus Messaging v1.3/README.md:54]:
```suggestion
💡 The EventHubs message pump and message router have a lot more options to fine tune the message handling to your needs. See [our official documentation](https://messaging.arcus-azure.net/Features/message-handling/event-hubs) for more information on this and other topics.
```

File: community/blog/2022-09 Added EventHubs support, correlated Service Bus messages, and more in Arcus Messaging v1.3/README.md [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+
+## Correlated Service Bus and EventHubs messages
+Service-to-service correlation is a big focus within Arcus. We have already updated our Web API package to fully support send/receive operations of correlated HTTP requests/responses. This results in a clear overview of all the made interactions in Application Insights. The same correlation functionality is now added to the Messaging library, making almost all our components correlation-ready.
```
Comment (fgheysels) [inline community/blog/2022-09 Added EventHubs support, correlated Service Bus messages, and more in Arcus Messaging v1.3/README.md:57]:
```suggestion
Service-to-service correlation is a big focus within Arcus. We have already updated our Web API package to fully support send/receive operations of correlated HTTP requests/responses. This results in a clear overview of all the interactions between multiple components in Application Insights. The same correlation functionality is now added to the Messaging library, making almost all our components correlation-ready.
```

File: community/blog/2022-09 Added EventHubs support, correlated Service Bus messages, and more in Arcus Messaging v1.3/README.md [hunk 5]
```diff
@@ -0,0 +1,129 @@
...
+
+Retrieving and processing messages via message pumps or message routers automatically results in request telemetry in Application Insights, so no consumer action has to be taken here. Placing messages on Service Bus or an EventHubs queue requires you to use one of our extensions on `ServiceBusSender` or `EventHubsProducerClient` respectively. Usually, when interacting with Azure in this way, you would register these clients into your application so they can be injected, reused and tested easily. [Azure SDK clients](https://www.nuget.org/packages/Microsoft.Extensions.Azure/) is a great way to do this.
```
Comment (fgheysels) [inline community/blog/2022-09 Added EventHubs support, correlated Service Bus messages, and more in Arcus Messaging v1.3/README.md:59]:
```suggestion
Retrieving and processing messages via message pumps or message routers automatically results in request telemetry in Application Insights, so no consumer action is required here. Placing messages on Service Bus or an EventHubs queue requires you to use one of our extensions on `ServiceBusSender` or `EventHubsProducerClient` respectively. Usually, when interacting with Azure in this way, you would register these clients into your application so they can be injected, reused and tested easily. [Azure SDK clients](https://www.nuget.org/packages/Microsoft.Extensions.Azure/) is a great way to do this.
```

File: community/blog/2022-09 Added EventHubs support, correlated Service Bus messages, and more in Arcus Messaging v1.3/README.md [hunk 6]
```diff
@@ -0,0 +1,129 @@
...
+
+## Removed exception details from TCP health probe
+We also added a small feature with a big security impact to this release. The v0.8 Arcus Templates release included a dedicated API health report model with the exception details removed. These details are useful for defect localization but are unnecessary information for the public eye. In certain cases, this information could be used maliciously and pose a security risk.
+
+The Arcus Messaging also has a variant on this health report endpoint, which is our TCP health probe that exposes the health of an application via a configurable TCP endpoint. This was using Microsoft's default health report, which included exception details.
+
+Starting from Arcus Messaging v1.3, these exception details are removed from the public endpoint which makes it a more safe and production-ready implementation.
+
+## Conclusion
+Arcus Messaging v1.3 is a big release for EventHubs but also provides extensive functionality to fully support service-to-service scenarios.
```
Comment (fgheysels) [inline community/blog/2022-09 Added EventHubs support, correlated Service Bus messages, and more in Arcus Messaging v1.3/README.md:123]:
```suggestion
Arcus Messaging v1.3 is a big release with support for EventHubs but also provides extensive functionality to fully support service-to-service scenarios.
```

**Review by fgheysels — APPROVED** [review]

File: community/blog/2022-09 Added EventHubs support, correlated Service Bus messages, and more in Arcus Messaging v1.3/README.md (hunk 3 above)
Comment (stijnmoreels) [reply community/blog/2022-09 Added EventHubs support, correlated Service Bus messages, and more in Arcus Messaging v1.3/README.md:54]: It's really influence. It has deserialization options that could even alter the message handler that could process the message.

Comment (fgheysels) [reply community/blog/2022-09 Added EventHubs support, correlated Service Bus messages, and more in Arcus Messaging v1.3/README.md:54]: What about modify ?

Comment (stijnmoreels) [reply community/blog/2022-09 Added EventHubs support, correlated Service Bus messages, and more in Arcus Messaging v1.3/README.md]: https://github.com/arcus-azure/arcus/pull/246/commits/40c47259b54adb87b3a55c4bf2eb327bbeb48f95

Comment (stijnmoreels) [reply community/blog/2022-09 Added EventHubs support, correlated Service Bus messages, and more in Arcus Messaging v1.3/README.md]: Changed it to 'modify'! 👍

**Pull request merged**